dts/bindings/sensor/ti,ina3221.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments INA3221 Triple-Channel Current/Power Monitor
+
+compatible: "ti,ina3221"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  conv-time-shunt:
+    type: int
+    default: 4
```

Comment:
The default value needs a justification in the description. Is it a register reset value?

Reply:
yes, added that in the description